docker-compose-prod.yml

```diff
@@ -0,0 +1,62 @@
+version: '2'
+services:
+  nginx:
+    image: jwilder/nginx-proxy
+    restart: always
+    ports: 
+      - "80:80"
+      - "443:443"
+    volumes:
+      - /var/run/docker.sock:/tmp/docker.sock:ro
+    logging:
+      driver: "json-file"
+      options: 
+        max-size: "2m"
+        max-file: "10"
+  stt:
+    container_name: app_stt
+    restart: always
+    build:
+      context: .
+      dockerfile: ./docker/Dockerfile-server-dev
+    ports:
+      - "8081:8081"
+    volumes:
+      - /var/log/stt/:/var/log/stt/
```

Comment:
I think this directory might need to be created explicitly prior to launch of the containers